settings.py

```diff
@@ -282,3 +284,21 @@
 MME_SLACK_ALERT_NOTIFICATION_CHANNEL = 'matchmaker_alerts'
 MME_SLACK_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_matches'
 MME_SLACK_SEQR_MATCH_NOTIFICATION_CHANNEL = 'matchmaker_seqr_match'
+
+#########################################################
+#  AnVIL Terra API specific settings
+#########################################################
+
+TERRA_API_ROOT_URL = 'https://api.firecloud.org/'
+
+GOOGLE_AUTH_CONFIG_DIR = os.environ.get('GOOGLE_AUTH_CONFIG_DIR', '')
+
+GOOGLE_AUTH_CLIENT_CONFIG = {}
+if GOOGLE_AUTH_CONFIG_DIR:
+    with open(os.path.join(GOOGLE_AUTH_CONFIG_DIR, 'client_secret.json'), 'r') as f:
+        GOOGLE_AUTH_CLIENT_CONFIG = json.load(f)
+
+GOOGLE_SERVICE_ACCOUNT_INFO = {}
+if GOOGLE_AUTH_CONFIG_DIR:
```

Comment:
you just need 1 if statement and can do both assignments in it

Reply:
Got it.